tests/test_decoder.py

```diff
@@ -142,6 +143,7 @@ def test_decoder_for_rule(self):
         self.assertEquals(decoded.alert_by, 'any')
         self.assertEquals(decoded.key, 'foo')
         self.assertEquals(len(decoded.conditions), 1)
+        self.assertEquals(decoded.status, 'logonly')
```

Comment:
If we're going to the trouble to provide constants, shouldn't the value decode to one of those constants as opposed to a string?